flow360/component/simulation/simulation_params.py

```diff
@@ -259,6 +260,12 @@ def check_consistency_ddes_volume_output(cls, v):
         """Only allow DDES output field when there is a corresponding solver with DDES enabled in models"""
         return _check_consistency_ddes_volume_output(v)
 
+    @pd.model_validator(mode="after")
+    @classmethod
```

Comment:
"after" model validators are not class methods. See : https://docs.pydantic.dev/latest/concepts/validators/#model-validators

Reply:
Thank you for the information. I have removed classmethod for the after model validator